### DISCUSSION TITLE
2주차 과제 - Interactive Portrait & Pattern Recognition

### DISCUSSION THREAD

**Discussion by youngsangcho** [youngsangcho/generativedesign2_2021#9] [Show and tell]
2주차 과제

**과제 1: Interactive Self-portrait**
> 마우스, 혹은 키보드에 반응하는 자화상 그리기
> 
> + rotate() 혹은 scale() 한 번 이상 사용.
> + // 코드를 설명하는 코멘트 최대한 많이 달기. (최소 3회)
> 
> + 1개 이상의 스크린샷 이미지
> + 스크린 동영상 캡쳐하여 유튜브 업로드 유튜브 embed
> + 간단한 설명 (왜 / 어떻게)
> + 코드 폴더 zip 하여 구글/네이버/드롭박스 등 클라우드 드라이브 링크
> + 코드 복사 붙여넣기
> 
> 제목 : 2주차 과제 - Interactive Portrait (혹은 자신이 붙인 제목)

**과제 2: Pattern Recognition**
> 내 생활, 머리 속의 (의미 있는) 패턴을 찾아 코드로 그림그리기
> 
> + for 구문을 이용하여 패턴을 그림.
> + // 코드를 설명하는 코멘트 최대한 많이 달기. (최소 3회)
> + noLoop() 사용해도 무방함.
> 
> + 1개 이상의 스크린샷 이미지
> + 간단한 설명 (왜 / 어떻게)
> + 코드 폴더 zip 하여 구글/네이버/드롭박스 등 클라우드 드라이브 링크
> + 코드 복사 붙여넣기
>
> 제목 : 2주차 과제 - Pattern Recognition (혹은 자신이 붙인 제목)

언제나 아래 형식에 맞춰 답글 달아주세요. (예시 참조)
[본인이름]
[링크 1]
[링크 2]

기한 : 2021.09.27 밤10시

**Comment by KimHyunJuhg** [comment]
김현정
https://blog.naver.com/khj317888/222517895339
https://blog.naver.com/khj317888/222517926785

**Comment by SeyounJung01** [comment]
정세윤 
1.https://blog.naver.com/seyounjj2001/222518761857
2.https://blog.naver.com/seyounjj2001/222518792250

**Comment by eunjinhong** [comment]
홍은진
1.https://blog.naver.com/tnekf1012/222518780252
2.https://blog.naver.com/tnekf1012/222518775485

**Comment by imyeeunok** [comment]
김예은

과제 1
https://blog.naver.com/010312kye/222518796410
과제2
https://blog.naver.com/010312kye/222518806806

**Comment by Joeun410** [comment]
김조은

과제1. https://blog.naver.com/joanne3456/222519112946

과제2. https://blog.naver.com/joanne3456

**Comment by youngsangcho** [comment]
지난주 와 같은 줌 링크에서 수업 진행합니다~
